Incremental/Incremental.swift

```diff
@@ -8,12 +8,11 @@ final class Queue {
     var processing: Bool = false
     
     func enqueue<S: Sequence>(_ edges: S) where S.Element: Edge {
-        self.edges.append(contentsOf: edges.map { ($0, $0.height) })
-        self.edges.sort { $0.1 < $1.1 }
```

Comment:
It looks like the `sort` went away by accident?

Reply:
It’s calling the method below it now. I figured that would be better since it has to map to the existential anyway.